scaling_performance/optimizing_storage.adoc

```diff
@@ -210,36 +233,13 @@ workloads.
 * All containers and images share the same pool of capacity. It cannot be resized
 without destroying and re-creating the pool.
 
-|Device Mapper Thin Provisioning
-|Also uses LVM, Device Mapper, and the dm-thinp kernel module. It differs by removing
-the loopback device, talking straight to a raw partition (no filesystem).
-a|* There are measurable performance advantages at moderate load and high density.
-* It gives you per-container limits for capacity (10G by default).
-a|* You have to have a dedicated partition for it.
-* It is not set up by default in Red Hat Enterprise Linux (RHEL).
-* All containers and images share the same pool of capacity. It cannot be resized
-without destroying and re-creating the pool.
-
-|OverlayFS
-|Combines a lower (parent) and upper (child) filesystem and a working directory
-(on the same filesystem as the child). The lower filesystem is the base image,
-and when you create new containers, a new upper filesystem is created
-containing the deltas.
-a|* Faster than Device Mapper at starting and stopping containers. The startup time
-difference between Device Mapper and Overlay is generally less than one second.
-* Allows for page cache sharing.
-|Not POSIX compliant.
 |===
 
-For more information about OverlayFS, including supportability and usage caveats, see the
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/?version=7[Red Hat Enterprise Linux (RHEL) 7 Release Notes] for your version.
-
-In production environments, using a Logical Volume Management (LVM) thin pool on top of regular block
-devices (not loop devices) for container images and container root file system
-storage is recommended.
+For better performance, Red Hat strongly recommends using the xref:benefits-of-using-the-overlay-graph-driver[overlayFS storage driver] over Device Mapper.
```

Comment:
I'd say "For better performance, use the xref:benefits-of-using-the-overlay-graph-driver[overlayFS storage driver] instead of Device Mapper. However, if you already use Device Mapper in a production environment, use thin provisioning for container images and container root file systems."

Reply:
@kalexand-rh I see that the Glossary of Terms and Conventions for Product Documentation suggests using _it is recommended_ for upstream/downstream (emphasis added):

_For better performance, **it is recommended** that you use the overlayFS storage driver instead of Device Mapper. However, if you are already using Device Mapper in a production environment, **it is recommended** that you use thin provisioning for container images and container root file systems._

Reply:
@mburke5678, I think the IBM Style guide makes more sense here: "recommend - Avoid if possible." :man_shrugging: Do what you think is best!